main.tf

```diff
@@ -1,3 +1,9 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {
```

Comment:
call it `default` sticking with our convention that if there's one of something, it's just called default

Reply:
Renamed